doc/source/ray-core/fault_tolerance/tasks.rst

```diff
@@ -27,6 +27,8 @@ field of the ``RayTaskError``.
   :start-after: __task_exceptions_begin__
   :end-before: __task_exceptions_end__
 
+If the user's exception cannot be serialized, it would be converted to a RayError:
```

Comment:
not sure why it ends with colon `:`.

and also maybe change to active tense, like:

````
If Ray cannot serialize the user's exception, it will convert the exception to a ``RayError``

````

Reply:
Thanks! Missed some sample code before. Fixed